testresultparser/junit_reports.go

```diff
@@ -3,115 +3,134 @@ package testresultparser
 import "encoding/xml"
 
 type report struct {
-    TestSuite   string     `xml:"name,attr"`
-    Tests       int        `xml:"tests,attr"`
-    Failures    int        `xml:"failures,attr"`
-    Errors      int        `xml:"errors,attr"`
-    Skipped     int        `xml:"skipped,attr"`
-    Time        string     `xml:"time,attr"`
-    TestResults []testCase `xml:"testcase"`
+	TestSuite   string     `xml:"name,attr"`
+	Tests       int        `xml:"tests,attr"`
+	Failures    int        `xml:"failures,attr"`
+	Errors      int        `xml:"errors,attr"`
+	Skipped     int        `xml:"skipped,attr"`
+	Time        string     `xml:"time,attr"`
+	TestResults []testCase `xml:"testcase"`
+	StdOut      systemOut  `xml:"system-out"`
```

Comment:
Tests for gradle 
``` go
Describe("Parsing Gradle default test results", func() {
	It("Should convert single test error to TestResult", func() 
```
and for Maven
``` go
Describe("Parsing Maven Surefire XML report", func() {
    It("should convert entire report", func()
``` 
tested the functionality.
Further added a separate test case to check for `standard output from the failure log`.